doc/time/strftime.rdoc

```diff
@@ -0,0 +1,353 @@
+
+The default width for a field is the width of the data itself.
+The integer optional minimum field width specifies the minimum width;
+it is ignored if smaller than the default width:
+
+  t = Time.now
+  t.strftime('%Y')   # => "2021"
+  t.strftime('%8Y')  # => "00002021"
+  t.strftime('%20Y') # => "2021"
+
+  t.strftime('%b')   # => "Oct"
+  t.strftime('%6b')  # => "   Oct"
+  t.strftime('%0b')  # => "Oct"
+
+<b>Flags</b>
+
+Flags for padding:
+
+- <tt>-</tt>: Omit numerical padding.
+- <tt>_</tt>: Use spaces for numerical padding.
+- <tt>0</tt>: Use zeros for numerical padding:
+
+Examples:
+
+  t.strftime('%-8Y')    # => "2021"
+  t.strftime('%_8Y')    # => "    2021"
+  t.strftime('%08Y')    # => "00002021"
+
+  t.strftime('%-6b')    # => "Oct"
+  t.strftime('%_6b')    # => "   Oct"
+  t.strftime('%06b')    # => "000Oct"
+
+Flags for casing:
+
+- <tt>^</tt>: Upcase.
+- <tt>%</tt>: Change case.
+
+Examples:
+
+  t.strftime('%p')  # => "PM"
+  t.strftime('%^p') # => "PM"
+  t.strftime('%#p') # => "pm"
+  t.strftime('%P')  # => "pm"
+  t.strftime('%^P') # => "PM"
+  t.strftime('%#P') # => "PM"
+
+<b>Modifiers</b>
+
+<em>Integer Modifier</em>
+
+Any conversion specifier may have an integer padding modifier added;
+for some specifiers and some integers, this adds padding;
+for others, the modifier is ignored.
+
+<em>Colon Modifiers</em>
+
+For specifier <tt>%z</tt> only, one, two, or three colons may added
+as a modifier.  See <tt>%z</tt> below.
+
+<em>Modifiers +E+ and +O+</em>
+
+For compatibility with other languages,
+some specifiers allow (but ignore) modifier +E+ or +O+;
+that is the directive behaves the same way with or without the modifier.
+
+In other cases, the modifier "spoils" the specifier,
+so that the text is not seen as a directive,
+but instead is passed through as plain text.
+
+<b>Conversion Specifiers</b>
+
+Note: Although technically a conversion specifier is a single character,
+we follow tradition here and list each with a prefixed percent character.
+
+<em>Year</em>
+
+- <tt>%Y</tt>: Year with century; may be negative; at least four digits.
+- <tt>%C</tt>: 2-digit century; <tt>year / 100</tt>, rounded down.
+- <tt>%y</tt>: 2-digit year; <tt>year % 100</tt>.
+
+Examples:
+
+  t = Time.now        # => 2021-11-01 09:14:14.0821024 -0500
+  t.strftime('%Y')    # => "2021"
+  t = Time.new(-1)    # => -0001-01-01 00:00:00 -0600
+  t.strftime('%Y')    # => "-0001"
+  t = Time.new(12345) # => 12345-01-01 00:00:00 -0600
+  t.strftime('%Y')    # => "12345"
+
+  t = Time.now        # => 2021-11-01 09:19:14.8958704 -0500
+  t.strftime('%C')    # => "20"
+  t = Time.new(2099)  # => 2099-01-01 00:00:00 -0600
+  t.strftime('%C')    # => "20"
+
+  t  = Time.now       # => 2021-11-01 09:21:41.0455507 -0500
+  t.strftime('%y')    # => "21"
+
+<em>Month</em>
+
+- <tt>%m</tt>: 2-digit month of the year, zero-padded; range (1..12).
+- <tt>%B</tt>: Full month name.
+- <tt>%b</tt>: 3-character abbreviated month name,
```

Comment:
Why did you remove some examples such as `%_m` and `%^B`?

I like the previous document that includes an example output in line, like `<tt>%B</tt>: Full month name. (``January'')`

Reply:
_ and ^ apply to many specifiers, not just these few.  I did add a note in Flags about multiple flags.

'example output in line':  There were 10 of these in all.  Will anyone misunderstand 'Full month name'?